src/account.rs

```diff
@@ -398,6 +403,19 @@ fn public_key_cli(wallet_path: &Path, account_ix: usize) -> Result<()> {
     Ok(())
 }
 
+//Prints the plain Address formatted pub key @{account_ix} (the one that doesn't start with 'fuel...')
+fn plain_address_cli(wallet_path: &Path, account_ix: usize) -> Result<()> {
+    let prompt =
+        format!("Please enter your password to display account {account_ix}'s public key: ");
+    let password = rpassword::prompt_password(prompt)?;
+    let secret_key = derive_secret_key(wallet_path, account_ix, &password)?;
+    let public_key = format!("{}", PublicKey::from(&secret_key));
+    let bech = Bech32Address::from_str(&public_key).expect("failed to create Bech32 address from String");
```

Comment:
Rather than using an `.expect`, let's try to use `?` so that error propagates to the end user rather than a `panic` with some context.